skin.json

```diff
@@ -91,6 +91,7 @@
     "showPromo": true
   },
   "pauseScreen": {
+    "autoHide": false,
```

Comment:
I think it might be more intuitive to refer to the controls autohiding instead of the pause screen. The property that currently controls the autohide is `controlBar.autoHide`, so this might be a bit inconsistent. That being said, not all of the controls are inside the control bar anymore, so `controlBar.autoHide` is a bit outdated.

Reply:
Should we create an entirely new ```autoHide``` property under a ```controls``` or ```skin``` key or something similar? I don't believe we should keep using ```controlBar.autoHide``` for the reasons you've stated.

Reply:
That would be ideal but the problem is that the change wouldn't be backward compatible

Reply:
I'll change it so that we use ```controlBar.autoHide```. We can discuss how to handle this moving forward at a later date

Reply:
Sounds good, although I think it needs to be a separate parameter so that the two states can be controlled independently

Reply:
Let's discuss with the playback team tomorrow